pkg/generator/vector/sources_test.go

```diff
@@ -0,0 +1,186 @@
+				Pipelines: []logging.PipelineSpec{
+					{
+						InputRefs: []string{
+							logging.InputNameApplication,
+						},
+						OutputRefs: []string{logging.OutputNameDefault},
+						Name:       "pipeline",
+					},
+				},
+			},
+			ExpectedConf: `
+# Logs from containers (including openshift containers)
+[sources.container_logs]
+  auto_partial_merge = true
+  exclude_paths_glob_patterns = ["/var/log/containers/vector-*_openshift-logging_*.log", "/var/log/containers/elasticsearch-*_openshift-logging_*.log", "/var/log/containers/kibana-*_openshift-logging_*.log"]
+`,
+		}),
+		Entry("Only Infrastructure", ConfGenerateTest{
+			CLFSpec: logging.ClusterLogForwarderSpec{
+				Pipelines: []logging.PipelineSpec{
+					{
+						InputRefs: []string{
+							logging.InputNameInfrastructure,
+						},
+						OutputRefs: []string{logging.OutputNameDefault},
+						Name:       "pipeline",
+					},
+				},
+			},
+			ExpectedConf: `
+# Logs from containers (including openshift containers)
+[sources.container_logs]
+  auto_partial_merge = true
+  exclude_paths_glob_patterns = ["/var/log/containers/vector-*_openshift-logging_*.log", "/var/log/containers/elasticsearch-*_openshift-logging_*.log", "/var/log/containers/kibana-*_openshift-logging_*.log"]
+
+[sources.journal_logs]
+  type = "journald"
+`,
+		}),
+		Entry("Only Audit", ConfGenerateTest{
+			CLFSpec: logging.ClusterLogForwarderSpec{
+				Pipelines: []logging.PipelineSpec{
+					{
+						InputRefs: []string{
+							logging.InputNameAudit,
+						},
+						OutputRefs: []string{logging.OutputNameDefault},
+						Name:       "pipeline",
+					},
+				},
+			},
+			ExpectedConf: `
+# Logs from host audit
+[sources.host_audit_logs]
+  type = "file"
+  ignore_older_secs = 600
+  include = ["/var/log/audit/audit.log"]
+
+# Logs from kubernetes audit
+[sources.k8s_audit_logs]
+  type = "file"
+  ignore_older_secs = 600
+  include = ["/var/log/kube-apiserver/audit.log"]
+
+# Logs from openshift audit
+[sources.openshift_audit_logs]
+  type = "file"
+  ignore_older_secs = 600
+  include = ["/var/log/oauth-apiserver.audit.log"]
+`,
+		}),
+		Entry("All Log Sources", ConfGenerateTest{
+			CLFSpec: logging.ClusterLogForwarderSpec{
+				Pipelines: []logging.PipelineSpec{
+					{
+						InputRefs: []string{
+							logging.InputNameApplication,
+							logging.InputNameInfrastructure,
+							logging.InputNameAudit,
+						},
+						OutputRefs: []string{logging.OutputNameDefault},
+						Name:       "pipeline",
+					},
+				},
+			},
+			ExpectedConf: `
+# Logs from containers (including openshift containers)
+[sources.container_logs]
+  auto_partial_merge = true
+  exclude_paths_glob_patterns = ["/var/log/containers/vector-*_openshift-logging_*.log", "/var/log/containers/elasticsearch-*_openshift-logging_*.log", "/var/log/containers/kibana-*_openshift-logging_*.log"]
+
+[sources.journal_logs]
+  type = "journald"
+
+# Logs from host audit
+[sources.host_audit_logs]
```

Comment:
I don't know if vector supports wild cards like fluent for routing but we should consider flipping the naming convention to something like "audit_logs_host".  This would allow us to filter from generic to specific